src/DssCdpManager.sol

```diff
@@ -237,14 +237,44 @@ contract DssCdpManager is DSNote {
         uint cdp,
         address dst
     ) public note cdpAllowed(cdp) urnAllowed(dst) {
-        address urn = urns[cdp];
-        (uint ink, uint art) = VatLike(vat).urns(ilks[cdp], urn);
+        (uint ink, uint art) = VatLike(vat).urns(ilks[cdp], urns[cdp]);
         VatLike(vat).fork(
             ilks[cdp],
-            urn,
+            urns[cdp],
             dst,
             toInt(ink),
             toInt(art)
         );
     }
+
+    // Import a position from org urn to the urn owned by cdp
+    function enter(
+        address org,
```

Comment:
Any reason for `org` instead of the more common `src`? Not that it matters much, just slightly more consistent IMO.

Reply:
you're right, just did the renaming